synthtool-venv/lib/python3.9/site-packages/synthtool/gcp/templates/python_library/.kokoro/samples/lint/presubmit.cfg

```diff
@@ -0,0 +1,6 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
```

Comment:
It looks like a site-packages folder was checked in for synthtool which I don't believe should be included.

Reply:
removed. thanks